app/views/users/index.html.erb

```diff
@@ -30,6 +31,7 @@
 	    <p><%= user.name %></p>
 			 <p> <%= image_tag user_avatar(user) , class: "img-fluid med-profile-img" %> </p>
 			 <%= button_to "Accept Friend",{:controller => 'friendships', :action => 'update',:user_id => current_user.id, :friend_id => user.id, :id => friendship.id }, {:method => :patch, :class => "bg-red font-white p-1 acceptingfriend"} %>
+			 <%= button_to "Reject Friend",{:controller => 'friendships', :action => 'destroy' :id => friendship.id }, {:method => :patch, :class => "bg-red font-white p-1 acceptingfriend"} %>
```

Comment:
Let's put a comma after `:action => 'destroy'` to fix a syntax error preventing the apllication from rendering Users page.